do_hindcasts.R

```diff
@@ -0,0 +1,57 @@
+library(tidyverse)
```

Comment:
I'm not a huge fan of this many library important due to namespace issues. Are we using anything other than `dplyr`?